apps/src/music/types.ts

```diff
@@ -15,6 +15,7 @@ export interface MusicLevelData extends ProjectLevelData {
   blockMode?: ValueOf<typeof BlockMode>;
   hideAiTemperature?: boolean;
   showAiTemperatureExplanation?: boolean;
+  showAiGenerateAgainHelp?: boolean;
```

Comment:
Probably not a realistic change now that our levels have been written, but it might be nice to group this under some `aiConfig: { hideAiTemperature: boolean; ... }` subgroup since we have multiple configuration flags now?

Reply:
Good point for a future tidy-up!